docs/process/roadmap.md

```diff
@@ -1,63 +1,71 @@
 # Roadmap
 
-The following are our marketing releases as planned to date. They contain the broad features we're considering. The further away a release is, the less defined it is. Releases are subject to change at any time.
+The following are our marketing releases, as planned to date (and we plan to revisit 6 months from now). They contain the broad features we're considering. The further away a release is, the less defined it is. Releases are subject to change at any time. This is not written in stone!!!
 
-## 1.1
+## 1.2 - Relationship between branches - lite
 
-- Relationship to _X_
-  - Show how the current branch relates to the target branch
-  - The target is the default branch or the upstream's default branch
-  - _Probably_ let the user switch the target branch
+- [Relationship between branches lite](https://github.com/desktop/desktop/issues/3956) - [#3956](https://github.com/desktop/desktop/issues/3956)
```

Comment:
We don't explain what "lite", "1.0" and "2.0" represent in terms of this document. I took a stab at this here, see if you like it or not:

> A feature may be suffixed with a label like "lite" or "1.0" or "2.0" - this is used to define the scope of the feature, as it is expected to evolve over time:
>
> - "lite" - the fundamentals of the feature are implemented, enough to get the common tasks done
> - "1.0" - the feature has evolved beyond the fundamentals, enough for every day use
> - "2.0" - with more knowledge about the domain and use cases, the feature now supports advanced tasks